src/json-to-postgres-table/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM node:alpine
+
+COPY ./package.json ./package-lock.json /home/
```

Comment:
these pathes will now need adjustment, see the other Dockerfiles for reference

Reply:
Thx. I updated it.